uw_sdbmyuw/__init__.py

```diff
@@ -28,8 +29,9 @@ def get_app_status(system_key):
         raise DataFailureException(url, response.status, response.data)
 
     if len(response.data) == 0:
-        raise Exception("%s Unexpected Response Data: %s" %
-                        (url, response.data))
+        is_cached = type(response) == restclients_core.models.MockHttp
```

Comment:
could you explain what this is checking?

Reply:
We're getting a bunch of throws of this exception in production (12.6k in the past week), and it would appear that it results from the SdbMyUW application returning an empty body. I'm curious as to whether this is the result of a bug in our cache, or it's an outage/bug in the SdbMyUW client application.